### ISSUE TITLE
Move postgres from docker to managed instance

### ISSUE THREAD

**Issue by willusds** [CDCgov/prime-central#154]
- [ ] Set up CI/CD

**Comment by pete-usds** [comment]
CICD implementation would be separate, either before or after, depending on when/how principals are set up. If we want this move done in this sprint we'd probably do it manually (unless CDC suddenly gets us the perms we need, then we'd want a separate ticket for CICD across the board).

**Comment by pete-usds** [comment]
I'm gonna take a crack at creating an interim migration process to hold us over until we move towards CICD.

**Comment by pete-usds** [comment]
@willusds @timothybest-usds FYSA I'm changing the content of this ticket to reflect that it should be to determine *how* we will migrate to a managed database and create a plan to do so for the beta/"prod" stack. 

The actual migration for the beta stack (and CICD implementation in the future) should be planned/managed separately once this work is complete.

**Comment by benwarfield-usds** [comment]
Just to get this visible to the more infrastructure-oriented folks: our current deploy does some things for the sake of convenience that we would prefer not to do in the longer term. If we can include stopping doing those things in this migration, it would be lovely, though we can also do it later:

* using the `public` schema instead of a custom schema, which means various things about default permissions that we have mostly not grappled with, because we are also
* using a single database user with both operations and maintenance powers, rather than one for the application with select/update and another for schema maintenance with create table/references etc.

The development setup actually does both of these things right, but all of our deployed environments are currently lazy about it.

**Comment by pete-usds** [comment]
Oh, that's definitely doable Ben - right now I'm literally doing a 1:1 port.

At this point I've successfully moved the staging database from a docker container to a hosted instance. The process is a bit ugly but it seems to work. I'm going to re-deploy everything and see if I can actually duplicate this feat from my notes though.

**Comment by pete-usds** [comment]
A repeatable if slightly manual (doesn't matter, only have to do it once) process has been worked out and tested with staging, documented in our internal repo. Whether or not y'all want to adjust it for changes is up to you, but this at least works in staging for the environment that's up right now.

https://github.com/usds/prime-simplereport-docs/blob/main/azure/improvised-stack/database-docker-to-managed.md

Moving this into QA for review by someone else and hopefully handoff for someone else to coordinate production migration (which should be a separate issue).

**Comment by timothybest-usds** [comment]
poking around https://staging.simplereport.org/app/organization/123/settings and it appears all the data is the same as prod

**Comment by pete-usds** [comment]
> poking around https://staging.simplereport.org/app/organization/123/settings and it appears all the data is the same as prod

It was cloned from prod at 1:07PM EST today, so should be the same as of then.

**Comment by timothybest-usds** [comment]
Migration plan look sensible as well
